pages/news/index.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div>
+    <v-container>
+      <NewsFilter :reloadNews="getNews" />
+      <v-row v-if="isNewsLoading">
+        <v-col v-for="index in 2" :key="index" class="col-12 col-md-6">
+          <NewsCardSkeleton />
+        </v-col>
+      </v-row>
+      <v-row v-else>
+        <v-col
+          v-for="(news, index) in newsData"
+          :key="`${news?.source.id}-${index}`"
+        >
+          <NewsCard :news="news" />
+        </v-col>
+      </v-row>
+    </v-container>
```

Comment:
Create a paginator up to 10 article at at ime

Reply:
resolved